grid/chat/workspaces/profile.supr

```diff
@@ -0,0 +1,170 @@
+"""
+Chat Workspaces
+Retrieve workspaces for the user.
+"""
+name = "chat/workspaces"
+version = "1.0.0"
+
+"""
+GetWorkspaces usecase
+"""
+usecase GetWorkspaces {
+  input {
+    """
+    As Bot
+    Boolean representing whether to authorize as bot
+    """
+    asBot boolean
+  }
+
+  result {
+    """
+    Workspaces
+    List of workspaces. Can be workspaces or guilds, depending on the provider capability.
+    """
+    workspaces! [Workspace]!
+
+    """
+    Rate Limit
+    Rate limit details.
+    """
+    rateLimit RateLimit
+  }
+
+  error Error
+
+  example Successful {
+    input {
+      asBot = true
+    }
+
+    result {
+      workspaces = [
+        {
+          id = '1'
+          createdAt = 1643798532134.069
+          name = 'workspace'
+          membersCount = 100
+        },
+        {
+          id = '2'
+          createdAt = 1643216443234.079
+          name = 'workspace 2'
+          membersCount = 4
+          description = 'Test workspace'
+        }
+      ]
+    }
+  }
+
+  example Failed {
+    input {}
+
+    error {
+      title = 'No workspaces found for current user.'
+    }
+  }
+}
+
+model Workspace {
```

Comment:
Shouldn't the workspace include some URL or something? Slack definitely has those and Discord servers have AFAIK vanity URLs (and if it's missing, I think the URL can be easily composed with server ID).

Reply:
I'll add optional field for it 👍🏻